tests/notifications-payload-guard.test.ts

```
@@ -0,0 +1,313 @@
+    safeParse: () => ({ success: true, data: {} }),
+  },
+}));
+
+void mock.module("@/utils/jwt", () => ({
+  createJwtToken: () =>
+    Promise.resolve("test-jwt-700-bytes-" + "x".repeat(680)),
+}));
+
+const { handleV2Notification } = await import(
+  "@/api/v2/notifications/handlers/webhook"
+);
```

ESLint

[error] Replace `·await·import(⏎··"`@/api/v2/notifications/handlers/webhook`"⏎` with `⏎··await·import("`@/api/v2/notifications/handlers/webhook`"`

(prettier/prettier)